packages/store/src/ResourceId.sol

```diff
@@ -45,4 +45,13 @@ library ResourceIdInstance {
   function getType(ResourceId resourceId) internal pure returns (bytes2) {
     return bytes2(ResourceId.unwrap(resourceId));
   }
+
+  /**
+   * @notice Get the name from a resource ID.
+   * @param resourceId The resource ID.
+   * @return A 30-byte name.
+   */
+  function getNameStore(ResourceId resourceId) internal pure returns (bytes30) {
```

Comment:
what about `getResourceName`?

Reply:
That's kinda better but the World name is also a "resource name" 🤔